Magefile.go

```diff
@@ -0,0 +1,127 @@
+//+build mage
+
+package main
+
+import (
+	"archive/zip"
+	"encoding/json"
+	"fmt"
+	"github.com/magefile/mage/mg"
+	"io"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+
+	// mage:import
+	"github.com/grafana/grafana-plugin-sdk-go/build"
+	"github.com/grafana/grafana-plugin-sdk-go/build/utils"
+)
+
+func CopyArtifacts() error {
+	// TODO: Only try to remove if dist exists, check error
+	_ = os.RemoveAll("dist")
+	if err := os.MkdirAll("dist", 0775); err != nil {
+		return err
+	}
+
+	fpaths := []string{"src/plugin.json", "README.md"}
+	fis, err := ioutil.ReadDir("img")
+	if err != nil {
+		return err
+	}
+	for _, fi := range fis {
+		fpaths = append(fpaths, path.Join("img", fi.Name()))
+	}
+
+	for _, fpath := range fpaths {
+		if err := utils.CopyFile(fpath, path.Join("dist", path.Base(fpath))); err != nil {
+			return err
+		}
+	}
+
+	return nil
+}
+
+func readPluginJSON() (map[string]interface{}, error) {
+	byteValue, err := ioutil.ReadFile(path.Join("src", "plugin.json"))
```

Comment:
```suggestion
func readDistPluginJSON() (map[string]interface{}, error) {
	byteValue, err := ioutil.ReadFile(path.Join("dist", "plugin.json"))
```
In the general case, the plugin.json in src will get modified... for the general function lets use dist